src/lib.rs

```diff
@@ -109,6 +112,16 @@ impl Builder {
         self
     }
 
+    /// If specified, a module with the given name will be generated which re-exports
+    /// all generated items.
+    ///
+    /// This is ignored by Prost, since Prost uses the package names of protocols
+    /// in any case.
+    pub fn package_name(&mut self, package_name: impl Into<String>) -> &mut Self {
```

Comment:
So there's no need of a getter?

Reply:
No, it's a builder API, it is only used internally when we generate the modules.